vendor/symfony-data/modules/default/templates/indexSuccess.php

```diff
@@ -0,0 +1,27 @@
+<meta http-equiv="refresh" content="0; url=http://corp.kaltura.com" />
+<!--?php decorate_with(sfLoader::getTemplatePath('default', 'defaultLayout.php')) ?>
```

Comment:
why not just remove the entire commented part?
there's no need for it, and better not have some picky pen-tester look at the 200 response body and say "it says symfony"

Reply:
@gonenradai done